internal/observability/client.go

```diff
@@ -64,6 +65,8 @@ func Init(ctx context.Context, serviceName, serviceVersion string, config Config
 	meterProvider := metric.NewMeterProvider(metric.WithReader(metric.NewPeriodicReader(otelMetricExporter)), metric.WithResource(res))
 	otel.SetMeterProvider(meterProvider)
 
+	observability.Init()
+
```

Comment:
The internal package should not depend on backend/controller/... things at all (or runner, etc.). This would create internal dependencies on `controller` and `runner`. Instead the `backend` stuff should use `internal` as needed.

Reply:
moved to `controller.go:New`